test/unit/models/classes/xml/XmlEditorTest.php

```diff
@@ -52,6 +58,9 @@ class XmlEditorTest extends TestCase
     /** @var FeatureFlagChecker|MockObject */
     private $featureFlagCheckerMock;
 
+    /** @var UserSettingsService|MockObject */
```

Comment:
*chore*: Lets rely on interface for open/closed implementation best practices

```suggestion
    /** @var UserSettingsServiceInterface|MockObject */
```